### PR TITLE
Add parameters to our kc template

### PR DESCRIPTION
**Pull request by hectorj2f**
related to: fabric8-services/fabric8-auth#45

ping @vpavlin: could you review this PR ? Thx.

### CODE REVIEW

**Comment by vpavlin** [comment]
I tested it locally and after fixing a bug in saasherder it worked as expected.

File: keycloak-services/keycloak.yaml [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
   path: /openshift/keycloak.app.yaml
   url: https://github.com/fabric8-services/keycloak-deployment
+  environments:
+  - name: production
+    parameters:
+      REPLICAS: 3
+      OPERATING_MODE: clustered
+  - name: staging
+    parameters:
+      REPLICAS: 3
```
Comment (hectorj2f) [inline keycloak-services/keycloak.yaml]: @vpavlin do I need it to be a string here as well ?

Comment (vpavlin) [inline keycloak-services/keycloak.yaml]: No, everything is converted to strings by calling subprocess in python, so this is fine

Comment (hectorj2f) [inline keycloak-services/keycloak.yaml]: awesome!

**Comment by hectorj2f** [comment]
@vpavlin we can merge if you want

**Pull request merged**